src/main/flight/mixer_tricopter.c

```diff
@@ -494,7 +418,7 @@ static void triTailTuneStep(servoParam_t *pServoConf, int16_t *pServoVal)
     }
 }
 
-static void tailTuneModeThrustTorque(struct thrustTorque_t *pTT, const bool isThrottleHigh)
+void tailTuneModeThrustTorque(thrustTorque_t *pTT, const bool isThrottleHigh)
```

Comment:
Yes agree completely. I rememberd I saw something like that but couldn't find it.